src/sections.js

```diff
@@ -12,6 +12,9 @@ const options = (section, context) => {
             const argument = arguments.getArgumentObject(object, context);
             context.options.push(argument);
         });
+        const delimiter = arguments.getDelimiterValue(
```

Comment:
I'm not sure, argument can have `delimiter` property and context delimiter can be determined by arguments delimiter